drivers/at86rf2xx/at86rf2xx_netdev.c

```diff
@@ -77,8 +77,8 @@ static int _init(netdev_t *netdev)
     gpio_set(dev->params.reset_pin);
     gpio_init_int(dev->params.int_pin, GPIO_IN, GPIO_RISING, _irq_handler, dev);
 
-    /* make sure device is not sleeping, so we can query part number */
-    at86rf2xx_assert_awake(dev);
```

Comment:
@smlng `at86rf2xx_reset` calls `at86rf2xx_hardware_reset` which results in either `P_ON` or `TRX_OFF` states, depending of the state before.